tensorboard/webapp/runs/views/runs_table/runs_data_table.ts

```diff
@@ -37,6 +37,7 @@ export class RunsDataTable {
   @Input() data!: TableData[];
   @Input() sortingInfo!: SortingInfo;
   @Input() experimentIds!: string[];
+  @Input() selectableColumns!: ColumnHeader[];
```

Comment:
Small Nit: I like potentialColumns better.  This name makes me think these are columns which have some selectable control or something. potentialColumns is also more consistent with the selector used.